RxAlamofire/Source/RxAlamofire.swift

```diff
@@ -809,13 +809,19 @@ extension Request {
     public func rx_progress() -> Observable<RxProgress> {
         return Observable.create { observer in
             self.progress() { bytesWritten, totalBytesWritten, totalBytesExpectedToWrite in
+                
                 observer.onNext(RxProgress(bytesWritten: bytesWritten, totalBytesWritten: totalBytesWritten, totalBytesExpectedToWrite: totalBytesExpectedToWrite))
+                
+                if totalBytesWritten == totalBytesExpectedToWrite {
```

Comment:
I would change this and check if `totalBytesWritten is >= totalBytesExpectedToWrite`, it might happen, it's an edge case, but we have to be safe on this matter.